go.mod

```diff
@@ -208,6 +209,8 @@ require (
 	k8s.io/klog/v2 v2.120.1
 )
 
+require github.com/open-telemetry/opentelemetry-collector-contrib/receiver/awscontainerinsightskueuereceiver v0.0.0-20241112181539-bc8278558352
```

Comment:
Can you move this up along with the rest?

Reply:
Done.